src/core/qgscoordinatereferencesystem.h

```diff
@@ -46,38 +46,189 @@ class QgsCoordinateReferenceSystem;
+ * QGIS comes with its internal database of coordinate reference systems (stored in SQLite) that
+ * allows lookups of CRS and seamless conversions between the various definitions.
+ *
+ * Most commonly one comes across two types of coordinate systems:
+ *
+ * 1. **Geographic coordinate systems** - based on a geodetic datum, normally with coordinates being
+ *    latitude/longitude in degrees. The most common one is World Geodetic System 84 (WGS84).
+ * 2. **Projected coordinate systems** - based on a geodetic datum with coordinates projected to a plane,
+ *    typically using meters or feet as units. Common projected coordinate systems are Universal
+ *    Transverse Mercator or Albers Equal Area.
+ *
+ * Internally QGIS uses proj4 library for all the math behind coordinate transformations, so in case
+ * of any troubles with projections it is best to examine the PROJ.4 representation within the object,
+ * as that is the representation that will be ultimately used.
+ *
+ * Methods that allow inspection of CRS instances include isValid(), authid(), description(),
+ * toWkt(), toProj4(), mapUnits() and others.
+ * Creation of CRS instances is further described in \ref crs_construct_and_copy section below.
+ * Transformations between coordinate reference systems are done using QgsCoordinateTransform class.
+ *
+ * For example, the following code will create and inspect "British national grid" CRS:
+ *
+ * ~~~{.py}
+ * crs = QgsCoordinateReferenceSystem("EPSG:27700")
+ * print "Valid:", crs.isValid()
+ * print "Description:", crs.description()
+ * print "PROJ.4:", crs.toProj4()
+ * ~~~
+ *
+ * This will produce the following output:
+ *
+ * ~~~
+ * Valid: True
+ * Description: OSGB 1936 / British National Grid
+ * PROJ.4: +proj=tmerc +lat_0=49 +lon_0=-2 +k=0.9996012717 +x_0=400000 +y_0=-100000 [output trimmed]
+ * ~~~
+ *
+ * CRS Definition Formats
+ * ======================
+ *
+ * This section gives an overview of various supported CRS definition formats:
+ *
+ * 1. **Authority and Code.** Also referred to as OGC WMS format within QGIS as they have been widely
+ *    used in OGC standards. These are encoded as `<auth>:<code>`, for example `EPSG:4326` refers
+ *    to WGS84 system. EPSG is the most commonly used authority that covers a wide range
+ *    of coordinate systems around the world.
+ *
+ *    An extended variant of this format is OGC URN. Syntax of URN for CRS definition is
+ *    `urn:ogc:def:crs:<auth>:[<version>]:<code>`. This class can also parse URNs (versions
+ *    are currently ignored). For example, WGS84 may be encoded as `urn:ogc:def:crs:OGC:1.3:CRS84`.
+ *
+ *    QGIS adds support for "USER" authority that refers to IDs used internally in QGIS. This variant
+ *    is best avoided or used with caution as the IDs are not permanent and they refer to different CRS
+ *    on different machines or user profiles.
+ *
+ *    See authid() and createFromOgcWmsCrs() methods.
```

Comment:
I tried to use it before, but that does not work well with when embedded in a list... it created a disruptive box that broke the flow of text in the HTML output